### PR TITLE
Review Downloads doc page

### PR DESCRIPTION
**Pull request by ritawatson**
Please review changes to the Downloads page for https://spectrocloud.atlassian.net/browse/DOC-397:

- Fixed URLs for MAAS PCG image and for OpenStack PCG image.
- Updated URL for vSphere PCG image to 1.3.0.
- Updated dates for all three URLs - please check that these are correct. The doc page showed 2022, but https://spectrocloud.atlassian.net/wiki/spaces/ENGINEERIN/pages/802947073/Golden+Images+List shows 2021. 

Screenshot of the page:

CleanShot 2022-11-08 at 15 55 21

### CODE REVIEW

**Review by karl-cardenas-coding — APPROVED** [review]
LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/972#pullrequestreview-1174212232" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/972#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
